packages/server-wallet/tsconfig.json

```diff
@@ -16,5 +16,5 @@
     {"path": "../nitro-protocol"},
     {"path": "../wallet-core"}
   ],
-  "include": ["src", "e2e-test"]
+  "include": ["src", "e2e-test", "deployment"]
```

Comment:
I'm not sure if we want this in here — we probably need a second config file...`tsconfig.integration.json` or something. Not that important really, but flagging it for now.